pkg/webservice/handlers.go

```diff
@@ -82,14 +82,23 @@ func getClusterInfo(w http.ResponseWriter, r *http.Request) {
 func getApplicationsInfo(w http.ResponseWriter, r *http.Request) {
 	writeHeaders(w)
 
+	queueName := r.URL.Query().Get("queue")
```

Comment:
the parameter must be sanity checked to make sure it is a queue:
* must be name <dot> name ...
* each name must comply with QueueNameRegExp (see the configs code for that)
* anything that does not comply give an error or be ignored